proxy/mitmproxy.py

```diff
@@ -91,7 +91,7 @@ def command(self):
                 "--save-stream-file",
                 self.path,
                 "--scripts",
-                os.path.join(self.scripts, "inject-deterministic.py"),
+                os.path.join(self.scripts, "record_scripts.py"),
```

Comment:
`--scripts` is a sequence of strings, could we list the individual scripts instead of having and extra script?